### PR TITLE
refactor: add rule to check if AsyncAPI document is defined or not

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Add rule to check if AsyncAPI document is defined or not.

cc @smoya 

**Related issue(s)**
Part of #481 
Part of #482

### CODE REVIEW

File: src/spectral.ts [hunk 1]
```diff
@@ -14,11 +18,49 @@ function configureRuleset(parser: Parser): RulesetDefinition {
...
+      'asyncapi-schemas-v2': aas2SchemaParserRule(parser),
       // We do not use these rules from the official ruleset due to the fact 
       // that the given rules validate only AsyncAPI Schemas and prevent defining schemas in other formats 
       'asyncapi-payload-unsupported-schemaFormat': 'off',
       'asyncapi-payload': 'off',
     },
   } as RulesetDefinition;
 }
+
+function asyncApi2IsAsyncAPI(): RuleDefinition {
```
Comment (smoya) [inline src/spectral.ts:31]:
```suggestion
function asyncApi2IsAsyncApi(): RuleDefinition {
```

Or the other way around, but to unify namings

Comment (magicmatatjahu) [reply src/spectral.ts:31]: Thanks!

**Comment by magicmatatjahu** [comment]
@smoya Done :) Please check again.

**Review by smoya — APPROVED** [review]
Greeeeeat!👏

**Pull request merged**